ioctl/client.go

```diff
@@ -46,9 +50,9 @@ type (
 		ReadSecret() (string, error)
 		// Execute a bash command
 		Execute(string) error
-		// doing
-		GetAddress(in string) (string, error)
-		// doing
+		// DefaultAddress returns default address if input empty
+		DefaultAddress(in string) (string, error)
```

Comment:
the name is still not clear. how about `AddressWithDefaultIfNotExist`

Reply:
ok